admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +180,173 @@ A failure from this check indicates that a significant amount of space in etcd
+----
+
+Alternatively, set any checks you want to disable as environment variables with
+`-e openshift_disable_check=<check1>,<check2>` when running the
+`ansible-playbook` command.
+
+[[admin-guide-health-checks-via-docker-cli]]
+=== Running Health Checks via Docker CLI
+
+It is possible a playbook may require dependencies that are not installed
+locally on the host running the `ansible-playbook` command. You can avoid this
+issue by running the *openshift-ansible* health checks directly from the
+ifdef::openshift-enterprise[]
+*ose-ansible*
+endif::[]
+ifdef::openshift-origin[]
+*origin-ansible*
+endif::[]
+image via the Docker CLI.
+
+To do so, specify your cluster's inventory file and the *_health.yml_* playbook
+when running the following `docker run` command as a non-root user that has
+privileges to run containers:
+
+----
+# docker run -u `id -u` \ <1>
+    -v $HOME/.ssh/id_rsa:/opt/app-root/src/.ssh/id_rsa:Z,ro \ <2>
+    -v /etc/ansible/hosts:/tmp/inventory:ro \ <3>
+    -e INVENTORY_FILE=/tmp/inventory \
+    -e PLAYBOOK_FILE=playbooks/byo/openshift-checks/health.yml \ <4>
+    -e OPTS="-v -e openshift_check_logging_index_timeout_seconds=45 -e etcd_max_image_data_size_bytes=40000000000" \ <5>
+ifdef::openshift-enterprise[]
+    openshift3/ose-ansible
+endif::[]
+ifdef::openshift-origin[]
+    openshift/origin-ansible
+endif::[]
+----
+<1> These options make the container run with the same UID as the current user,
+which is required for permissions so that the SSH key can be read inside the
+container (SSH private keys are expected to be readable only by their owner).
+<2> Mount SSH keys as a volume under *_/opt/app-root/src/.ssh_* under normal usage
+when running the container as a non-root user.
+<3> Change *_/etc/ansible/hosts_* to the location of your cluster's inventory file,
+if different. This file will be bind-mounted to *_/tmp/inventory_*, which is
+used according to the `INVENTORY_FILE` environment variable in the container.
+<3> The `PLAYBOOK_FILE` environment variable is set to the location of the
+*_health.yml_* playbook relative to *_/usr/share/ansible/openshift-ansible_*
+inside the container.
+<4> Set any variables desired for a single run with the `-e key=value` format.
+
+In the above command, the SSH key is mounted with the `:Z` flag so that the
+container can read the SSH key from its restricted SELinux context; this means
+that your original SSH key file will be relabeled to something like
+`system_u:object_r:container_file_t:s0:c113,c247`. For more details about `:Z`,
+see the `docker-run(1)` man page.
+
+Keep this `:Z` use in mind when providing these volume mount specifications
+because this could have unexpected consequences. For example, if you mount (and
+therefore relabel) your *_$HOME/.ssh_* directory, you will block *sshd*
+from accessing your keys. This is a reason why you might want to work on a
+separate copy of the SSH key, so that the original file's labels remain
+untouched.
```

Comment:
I would reword this just a bit to be clearer about the problem and stronger in the suggested remedy.

> Keep this in mind for these volume mount specifications ￼because it could have unexpected consequences. For example, if you mount (and ￼therefore relabel) your *_$HOME/.ssh_* directory, *sshd* will become unable to access your public keys to allow remote login. To avoid altering the original file labels, mounting a copy of the SSH key (or directory) is recommended.